maintenance/rebuildLocalisationCache.php

```diff
@@ -151,6 +158,28 @@ function ( $item ) use ( $firstCodes ) {
 	}
 
 	/**
+	 * Helper function to include all localisation files to localisation cache rebuild process.
+	 * It looks for i18n and alias files from whole code directory and registers them in global array.
+	 */
+	private function includeAllMessagesFiles() {
```

Comment:
Make sure, the method follows MW coding conventions. Namely, it's all about spaces after ('s and before )'s, as they are present in the above `execute()` methods.